### PR TITLE
📝 docs: Fix mentions of `SSO_PROVIDERS` and `NEXTAUTH_SECRET`

### PR DESCRIPTION
**Pull request by johnnymo87**
#### 💻 变更类型 | Change Type

- [ ] ✨ feat
- [ ] 🐛 fix
- [ ] ♻️ refactor
- [ ] 💄 style
- [ ] 🔨 chore
- [ ] ⚡️ perf
- [x] 📝 docs

#### 🔀 变更说明 | Description of Change

I have a problem where my SSO login option disappeared from my Vercel deploy after I deployed the ~100 commits that I had lagged behind.

Given what changed in https://github.com/lobehub/lobe-chat/pull/2983, I think mentions of `SSO_PROVIDERS` and `NEXTAUTH_SECRET` in the docs are wrong? I think they should be `NEXT_AUTH_SSO_PROVIDERS` and `NEXT_AUTH_SECRET` respectively?

Not sure, still trying to figure this out.

EDIT: In my vercel deploy of lobe-chat, I tested out switching to these environment variables, and this fixed my problem. My SSO login is back 👍

### CODE REVIEW

File: docs/self-hosting/advanced/sso-providers/auth0.mdx [hunk 1]
```diff
@@ -77,8 +77,8 @@ When deploying LobeChat, you need to configure the following environment variabl
 | Environment Variable | Type | Description |
 | --- | --- | --- |
 | `ENABLE_OAUTH_SSO` | Required | Enable single sign-on (SSO) for LobeChat. Set to `1` to enable single sign-on. |
```
Comment (cy948) [inline docs/self-hosting/advanced/sso-providers/auth0.mdx]: Hey, `ENABLE_OAUTH_SSO` has been deprecate, just delete it. And did you think is a good idea to delete `SSO_PROVIDERS ` and `NEXTAUTH_SECRET ` in the sso provider pages and only keep this variables on the [auth page](https://lobehub.com/docs/self-hosting/advanced/authentication)?

Comment (johnnymo87) [reply docs/self-hosting/advanced/sso-providers/auth0.mdx]: > Hey, ENABLE_OAUTH_SSO has been deprecate, just delete it.

Addressed in [Delete ENABLE_OAUTH_SSO](https://github.com/lobehub/lobe-chat/pull/3018/commits/6b47d4f2d9b8960d7b1bdb575ea933d1296e426e) 👍 

> Do you think is a good idea to delete `SSO_PROVIDERS` and `NEXTAUTH_SECRET` in the sso provider pages and only keep this variables on the [auth page](https://lobehub.com/docs/self-hosting/advanced/authentication)?

Is your suggestion that users should read the general auth page for the "core" set of environment variables, and then they should read their SSO-specific page for any SSO-specific environment variables? That does sound like a better way to keep things organized and up to date 👍  But maybe that would be better as a separate PR, one that looks specifically for all the different ways these SSO-specific pages overlap with the general auth page?

**Review by arvinxx — APPROVED** [review]
Thanks for your contribution~

File: docs/self-hosting/advanced/sso-providers/auth0.mdx (hunk 1 above)
Comment (cy948) [reply docs/self-hosting/advanced/sso-providers/auth0.mdx]: Yes, your understanding is very accurate, looking forward to your next PR.

**Pull request merged**